src/actions/braze/braze.ts

```diff
@@ -96,14 +101,24 @@ export class BrazeAction extends Hub.Action {
     let totalCount = 0
     let fieldlist: Hub.Field[] = []
     let bzIdField = ""
+    const bzAttributeFields: BrazeTags[] = []
     let rows: BrazeApiRow[] = []
+    const bzRegExp = new RegExp(BrazeConfig.BRAZE_ATTRIBUTE_REGEX, "gi")
     try {
       await request.streamJsonDetail({
         onFields: (fields) => {
           fieldlist = Hub.allFields(fields)
           for (const field of fieldlist) {
-            if (field.tags  && field.tags.find((tag: string) => tag === BrazeConfig.BRAZE_ID_TAG )) {
-              bzIdField = field.name
+            if (field.tags) {
```

Comment:
nit - should we use a guard clause instead?
https://blog.codinghorror.com/flattening-arrow-code/
```suggestion
            if (!field.tags) {
              continue
            }
```

Reply:
i don't have a specific preference so this has been changed.